docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -106,89 +141,154 @@ For more information about `missing_field_count`,
 see the {ref}/ml-get-job-stats.html[get {anomaly-job} statistics API].
 
 
-[float]
-== Terms aggregation size affects data analysis
-//See x-pack-elasticsearch/#601
+[discrete]
+=== Security integration
 
-By default, the `terms` aggregation returns the buckets for the top ten terms.
-You can change this default behavior by setting the `size` parameter.
+When the {es} {security-features} are enabled, a {dfeed} stores the roles of the 
+user who created or updated the {dfeed} **at that time**. This means that if the 
+roles the user has are changed after they create or update a {dfeed} then the 
+{dfeed} continues to run without change. However, if instead the permissions 
+associated with the roles that are stored with the {dfeed} are changed then this 
+affects the {dfeed}. For more information, see <<ml-datafeeds>>.
 
-If you are send pre-aggregated data to a job for analysis, you must ensure
-that the `size` is configured correctly. Otherwise, some data might not be
-analyzed.
 
-[float]
-== Fields named "by", "count", or "over" cannot be used to split data
-//See x-pack-elasticsearch/#858
+[discrete]
+[[ml-result-size-limitations]]
+=== Job and {dfeed} APIs have a maximum search size
+//https://github.com/elastic/elasticsearch/issues/34864
 
-You cannot use the following field names in the `by_field_name` or
-`over_field_name` properties in a job: `by`; `count`; `over`. This limitation
-also applies to those properties when you create advanced jobs in {kib}.
+In 6.6 and later releases, the {ref}/ml-get-job.html[get jobs API] and the
+{ref}/ml-get-job-stats.html[get job statistics API] return a maximum of 10,000
+jobs. Likewise, the {ref}/ml-get-datafeed.html[get {dfeeds} API] and the
+{ref}/ml-get-datafeed-stats.html[get {dfeed} statistics API] return a maximum of
+10,000 {dfeeds}.
+
+
+[discrete]
+[[ml-forecast-limitations]]
+=== Forecast limitations
```

Comment:
Suggest we create two forecast limitations... 1) forecast operational limitations and 2) Unsupported forecast configuration options.

Move `over_field_name` and the unsupported functions over to configuration and leave the operational limitations mostly untouched otherwise.

Reply:
Thanks! Addressed it via 6a09b63. Also consolidated the intro of forecast operational limitations.